test/objects.js

```diff
@@ -681,6 +681,67 @@
     assert.ok(_.isBoolean(false), 'and so is false');
   });
 
+  QUnit.test('isMap', function(assert) {
+    var keyString = 'a string', keyObj = {}, keyFunc = function() {};
+    var obj = new Map([[keyString, 'value one'], [keyObj, 'value two'],
+      [keyFunc, 'value three']]);
+
```

Comment:
No prob. There's a bit more it can come after the other things are addressed.
You can use the travis checks indicator to know when issues with tests have been resolved.
![image](https://cloud.githubusercontent.com/assets/4303/13643916/07498df4-e5d8-11e5-9ede-d1019e44c25b.png)